### PR TITLE
kadanes algorithm code in java

### PR DESCRIPTION
**Pull request by ghost**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

this code is for Kadane's Algorithm its a beginner-friendly code that can be understood by a beginner also 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Java/Algorithms/kadanes_Java.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package DsAlgo;
+import java.util.*;
+public class kadanes_Java {
```
Comment (iamrajiv) [inline Java/Algorithms/kadanes_Java.java]: @HarshAgarwal-DEV Use proper naming conventions for Classes, `public class Kadane` will work fine.

File: Java/Algorithms/kadanes_Java.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+			}
+			if(curSum>maxSoFar) {
+				maxSoFar=curSum;
+			}
+		}
+		System.out.println(maxSoFar);
+		
+		sc.close();
+	}
+
```
Comment (iamrajiv) [inline Java/Algorithms/kadanes_Java.java:5]: @HarshAgarwal-DEV remove unnecessary lines.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@iamrajiv please review it once again

File: Java/README.md [hunk 3]
```diff
@@ -16,6 +16,8 @@
 * [Segment Tree](ds/SegmentTree.java)
 * [Stack](ds/Stackll.java)
 
+## Algorithms
+*[Kadane's Algorithm](Algorithms/kadanes_Java.java)
```
Comment (iamrajiv) [inline Java/README.md]: @HarshAgarwal-DEV give space after `*`

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Java/README.md (hunk 3 above)
Comment (ghost) [reply Java/README.md]: done with that @iamrajiv

**Review by iamrajiv — APPROVED** [review]

**Comment by iamrajiv** [comment]
@HarshAgarwal-DEV LGTM! Thanks for the PR.

**Comment by iamrajiv** [comment]
@all-contributors please add @HarshAgarwal-DEV for code

**Comment by iamrajiv** [comment]
@all-contributors please add @HarshAgarwal-DEV for code

**Pull request merged**